CONTRIBUTING.md

```diff
@@ -9,3 +9,8 @@ $ gofmt -w ./
 $ staticcheck ./...
 $ go test -race ./...
 ```
+
+## Pre-commit hook
+
+Whenever you commit some changes to this repository, our pre-commit hook will automatically
```

Comment:
Could you please add the instruction `git config core.hooksPath .githooks`? The user needs to execute this command, so the pre-commit hooks trigger.

Would be great if you could give it a try :D

Reply:
On that topic. I try to keep text as minimum as possible (not hiding information, but being straightforward. On that, I would suggest just a simple sentence _with a call to action or instruction, e.g.:

> There is a pre-commit hook to run all these commands for you once you commit:
>
> ```console
>$ git config etc
> ```

Reply:
@cuducos can you check if the text is better now?

Reply:
Totally not a blocker, but I think it's more verbose than necessary. I don't think we need a section for that either. It's just a way to automate the three commands listed above.

1. What do we need? Contributors to use the hook. 
2. How do they use it? Running a single command.
3. Why would they use it? Because it's damn easy.

You do not have to follow exactly these words, but I think we can be shorter and have more chances people will read it. In sum, my diff for this file would simply be (please, note I removed the heading you suggested):

```diff
+
+ To make it easy for you, we have pre-commit hooks set up:
+ 
+ ```console
+ $ git config core.hooksPath .githooks
+ ```
```